<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDCountersTotal</class>
 <widget class="QMainWindow" name="WBSDCountersTotal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counters Total</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>462</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item row="14" column="1">
      <widget class="Line" name="line_32">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="Line" name="line_33">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="Line" name="line_34">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="Line" name="line_35">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="Line" name="line_36">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="Line" name="line_37">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="Line" name="line_38">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="Line" name="line_39">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="Line" name="line_40">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="1">
      <widget class="Line" name="line_43">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="32" column="1">
      <widget class="Line" name="line_44">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="28" column="1">
      <widget class="Line" name="line_42">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="26" column="1">
      <widget class="Line" name="line_41">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="Line" name="line_45">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="22" column="2">
      <widget class="Line" name="line_46">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="24" column="2">
      <widget class="Line" name="line_47">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="26" column="2">
      <widget class="Line" name="line_48">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="28" column="2">
      <widget class="Line" name="line_49">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="2">
      <widget class="Line" name="line_50">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="32" column="2">
      <widget class="Line" name="line_51">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="Line" name="line_18">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="Line" name="line_17">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="Line" name="line_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="Line" name="line_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="Line" name="line_30">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="Line" name="line_31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="28" column="0">
      <widget class="Line" name="line_22">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="26" column="0">
      <widget class="Line" name="line_21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="27" column="2">
      <widget class="QLabel" name="instantVolNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Instant volume</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Right BIB volume</string>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QLabel" name="rightBIB">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Instant volume non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="leftBIB">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="lblUnit0">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="lblUnit1">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLabel" name="lblUnit3">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="lblUnit4">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="QLabel" name="lblUnit5">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
     <item row="27" column="1">
      <widget class="QLabel" name="lblUnit6">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
     <item row="29" column="0">
      <widget class="QLabel" name="lb_overall">
       <property name="text">
        <string>total overall counter since reset</string>
       </property>
      </widget>
     </item>
     <item row="29" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="31" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>total overall counter non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="31" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="31" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="Line" name="line_29">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="totalSmall">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Total Small drinks non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Total drinks non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="29" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Left BIB volume</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Total Small drinks</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Total Large drinks non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="totalLarge">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="totalDrinks">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Total Medium drinks non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Total Large drinks</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Total Medium drinks</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total drinks</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="totalMedium">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="totalMediumNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="totalSmallNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="totalDrinksNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="25" column="2">
      <widget class="QLabel" name="rightBIBNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Right BIB volume non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="23" column="2">
      <widget class="QLabel" name="leftBIBNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="21" column="2">
      <widget class="QLabel" name="instantVol">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Left BIB volume non-resetable</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="Line" name="line_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="Line" name="line_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="Line" name="line_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="32" column="0">
      <widget class="Line" name="line_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="Line" name="line_25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="0">
      <widget class="Line" name="line_23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="totalLargeNoRe">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="Line" name="line_27">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="Line" name="line_28">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="Line" name="line_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="Line" name="line_26">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
